src/xenia/gpu/trace_viewer.cc

```diff
@@ -408,6 +408,12 @@ void TraceViewer::DrawPacketDisassemblerUI() {
         }
         break;
       }
+      case TraceCommandType::kRegisters: {
```

Comment:
Oops, didn't notice them, will fix by myself possibly, thank you! These are used for internal purposes (not a part of guest packets — so no need to display them visually), and need to be handled like kMemoryWrite or kEdramSnapshot.